.github/workflows/benchmark_nightly_cpu.yml

```
@@ -0,0 +1,25 @@
+name: Benchmark torchserve cpu nightly
+
+on:
+  # run every day at 2:15am
+  schedule:
+    - cron:  '15 02 * * *'
+jobs:
+  nightly:
+    runs-on: self-hosted
+    steps:
+      - name: Setup Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+          architecture: x64
+      - name: Checkout TorchServe
+        uses: actions/checkout@v2
+      - name: Install dependencies
+          run: |
+            apt-get update
+            apt-get install sudo -y
+            sudo apt-get install apache2-utils
+            pip install -r benchmarks/requirements-ab.txt
+      - name: Benchmark cpu nightly
+        run: python benchmarks/auto_benchmark.py --input benchmarks/benchmark_config_cpu.yaml --skip false
```

Save artifacts to github as well since not everyone will have access to S3 bucket
```suggestion
        run: python benchmarks/auto_benchmark.py --input benchmarks/benchmark_config_cpu.yaml --skip false
     - name: Save benchmark artifacts
        uses: actions/upload-artifact@v2
        with:
          name: nightly artifact
          path: /tmp/ts_benchmark 
```